apps/daimo-mobile/src/view/screen/DepositScreen.tsx

```diff
@@ -138,15 +141,47 @@ function LandlineAccountList() {
   );
 }
 
+function getBinanceOpener(
+  account: Account,
+  setProgress: (progress: Progress) => void
+) {
+  const rpcFunc = getRpcFunc(daimoChainFromId(account.homeChainId));
+
+  const platform = ["ios", "android"].includes(Platform.OS)
+    ? (Platform.OS as PlatformType)
+    : "other";
+
+  if (platform === "other") {
+    return null;
+  }
+
+  return async () => {
+    setProgress("loading");
+    const url = await rpcFunc.getBinanceWithdrawalURL.query({
+      addr: account.address,
+      platform,
+    });
```

Comment:
does this withdrawal URL have to be generated right before use, or can it be generated ahead of time? like the `recommendedExchanges` in sync

Reply:
see the comment 20 lines down